x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/process_descendant_tooltip.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiToolTip, EuiText, EuiIcon } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { PROCESS_DESCENDANT_EVENT_FILTER_EXTRA_ENTRY_TEXT } from '../../../../../../common/endpoint/service/artifacts/constants';
+
+interface ProcessDescendantsTooltipProps {
+  testIdPrefix?: string;
+  indicateExtraEntry?: boolean;
+}
+
+export const ProcessDescendantsTooltip = ({
```

Comment:
please use `memo()` on this component definition.

Reply:
dc52be01304

Reply:
dc52be01304af7bc0a3ea67d787f6c29f6d55e46